### ISSUE TITLE
help with query

### ISSUE THREAD

**Issue by TanyaNikitina** [KatyaAplonova/ReportedDiscourseProject#20]
Is there a way to search for words with empty annotations in ps or ge?

**Comment by rvoll** [comment]
Yes. You can tick off regular expressions in the search field, choose the tier in which you want to search and enter 
^$ 
in the search field. This should give you all empty annotations on this tier.

**Comment by TanyaNikitina** [comment]
Thank you, this is extremely helpful! It turns out that some annotations went missing when I exported from Toolbox, this will help me fix it.

**Comment by IndianaTones** [comment]
I created a new folder for ELAN queries: https://github.com/KatyaAplonova/ReportedDiscourseProject/tree/master/ELAN-queries
You can use these by going to Search...Structured Search Multiple eaf or CorpA Multiple Files Search and clicking Load Query at the upper right hand corner of the screen that appears after you have created the domain (folder) of files you want to search.

**Comment by TanyaNikitina** [comment]
Thank you, it really helps!

**Comment by IndianaTones** [comment]
One open question is how to export only the content of the Discourse Reports. Thus far, this is not straightforward. One can search `Discourse Report\b` on tiers of the type `rp`, but because this tier is not a 'child' nor is it a 'parent', of the `tx`type tier, I do not know of a way to export only the content of that lies within the report. Hope this makes sense; maybe others among you have tried and/or found a solution?
